sympy/solvers/inequalities.py

```diff
@@ -518,7 +519,11 @@ def solve_univariate_inequality(expr, gen, relational=True, domain=S.Reals, cont
                 solns = solvify(e, gen, domain)
                 if solns is None:
                     # in which case we raise ValueError
-                    raise ValueError
+                    try:
+                        e = orig_expr.lhs - orig_expr.rhs
+                        return set((Poly(e, _gen).all_roots()))
+                    except PolynomialError:
+                        raise ValueError
```

Comment:
Can we pass a message as well to `ValueError`?

Reply:
Is it possible to remove try-expect and use if-else?

Reply:
Logically, why catch an exception and throw another exception. The possible use case that I can foresee is using exception as a form of message passing to the caller. Why not throw the same exception that you have caught.

Reply:
I believe that passing a message inside `ValueError` can help the user in taking a corrective action easily.

Reply:
@ishanaj Can you please address https://github.com/sympy/sympy/pull/13975#discussion_r324988974